webui/tests/tests/StepImplementation.ts

```diff
@@ -136,6 +142,28 @@ export default class StepImplementation {
     }
   }
 
+  /* Browser Utility Steps */
+
+  @Step('Should not have element <selector> present')
+  public async hasNoElement(selector: string) {
+    const exists = await $(selector).exists(1000);
+    assert.ok(!exists);
+  }
+  @Step('Should have element <selector> present')
+  public async hasElement(selector: string) {
+    await $(selector).exists();
+  }
+
+  @Step('Switch to mobile view')
```

Comment:
use https://docs.taiko.dev/api/emulatedevice/ instead?

Reply:
I checked their device list and it seems to only have mobile and table devices, so we would still need to have a desktop viewport defined. I think what you have setup looks good.

Reply:
I take that back, for just the mobile view, there's a lot more info than just the width and height. `iPhone 7` (most popular mobile device) via `emulateDevice` sounds good
```
  'iPhone 7 Plus': {
    userAgent:
      'Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1',
    viewport: {
      width: 414,
      height: 736,
      deviceScaleFactor: 3,
      isMobile: true,
      hasTouch: true,
      isLandscape: false,
    },
  }
```